### DISCUSSION TITLE
Use SQS XML API

### DISCUSSION THREAD

**Discussion by martinssipenko** [aws/aws-sdk-php#2864] [Q&A] [answered]
Is there a way to configure SDK to use older XML SQS API instead of new JSON api?

**Comment by yenfryherrerafeliz** [comment, marked as answer]
Hi @martinssipenko, you can use the version [3.277.2](https://github.com/aws/aws-sdk-php/releases/tag/3.277.2), which is the version before support for JSON was added to the SDK. The only thing about this is that you will not be able to have the latest changes and fixes of the SDK.

I hope this help you.

Thanks!
